virttest/qemu_vm.py

```diff
@@ -3231,8 +3231,6 @@ def update_vga_global_default(self, params, migrate=None):
             help_info = process.run(help_cmd, shell=True, verbose=False).stdout_Text
             for pro in re.findall(r"%s.(\w+)=" % vga_type, help_info):
                 key = [vga_type.lower(), pro]
-                if migrate:
-                    key.append("dst")
```

Comment:
Seems we have to keep this for avoiding regressions.

Reply:
I deleted it because I cannot see any usage in the test provider, and if it's present, the `define` method requires the `migrate` parameter, this is inconsistent with our needs. I know this is a break change, do you have ideas to handle it?